src/identity/contactMapping.ts

```diff
@@ -195,7 +200,7 @@ export function* fetchAddressesAndValidateSaga({
     yield put(endFetchingAddresses(e164Number, true))
     ValoraAnalytics.track(IdentityEvents.phone_number_lookup_complete)
   } catch (error) {
-    Logger.error(TAG + '@fetchAddressesAndValidate', `Error fetching addresses`, error)
```

Comment:
i assume it's okay to downgrade to logger.debug here, since we will be able to monitor the lookupPhoneNumber service and it's not a unrecoverable error